robot/web/web_test.go

```diff
@@ -589,6 +649,87 @@ func TestWebWithBadAuthHandlers(t *testing.T) {
 	test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
 }
 
+func TestWebWithOnlyNewAPIKeyAuthHandlers(t *testing.T) {
+	logger := golog.NewTestLogger(t)
+	ctx, injectRobot := setupRobotCtx(t)
+
+	svc := web.New(injectRobot, logger)
+
+	options, _, addr := robottestutils.CreateBaseOptionsAndListener(t)
+	apiKeyID1 := "6a948-5094-4f36-aae1-8f41fc15907a"
+	apiKey1 := "fvauz937m2xhcq2vtov91wz1bretzg2k"
+	apiKeyID2 := "4a948-5094-4f36-aae1-8f41fc15907b"
+	apiKey2 := "gsquz817m2xhcq2vtov91wz1bretzg2k"
```

Comment:
is there a way we could write a helper function that would help us generate random strings like this for an api key?

Reply:
updated to use same functions we use to generate these keys in `app`. good idea!

Reply:
Nice :)